index.html

```diff
@@ -526,7 +526,7 @@ <h2 id="how" class="title is-2">How<a class="anchor-link" href="#how">#</a></h2>
                     <p>
                         Here, we are using a Python one-liner to encode our diagram using deflate + base64:
                     </p>
-                    <pre><code class="language-sh">cat hello.dot | python -c "import sys; import base64; import zlib; print(base64.urlsafe_b64encode(zlib.compress(sys.stdin.read(), 9)))"</code></pre>
+                    <pre><code class="language-sh">cat hello.dot | python2 -c "import sys; import base64; import zlib; print(base64.urlsafe_b64encode(zlib.compress(sys.stdin.read(), 9)))"</code></pre>
```

Comment:
Didn't test on Python2 but the following is working on Python3:

```suggestion
                    <pre><code class="language-sh">cat hello.dot | python -c "import sys; import base64; import zlib; print(base64.urlsafe_b64encode(zlib.compress(sys.stdin.read().encode('utf-8'), 9)).decode('ascii'))"</code></pre>
```

Reply:
I can confirm that this command works on both Python 2 and Python 3.
@jamietanna Could you please update this pull request?